### PR TITLE
MV3 Content security policy

### PR DESCRIPTION
**Pull request by rebloor**
#### Summary
Updates to document the Manifest V3 changes to the content_security_policy manifest key along with related changes to the content security policy article.

#### Metadata

This PR…

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md [hunk 1]
```diff
@@ -59,18 +118,40 @@ There are restrictions on the policy you can specify here:
 
 Require that all types of content should be packaged with the extension:
 
+Allow remote scripts from "https\://example.com":
```
Comment (MrBrain295) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:121]:
```suggestion
Allow remote scripts from "https://example.com":
```

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md [hunk 2]
```diff
@@ -50,6 +50,65 @@ There are restrictions on the policy you can specify here:
...
+  <tbody>
+    <tr>
+      <td>
+        <code>content_scripts</code>
+      </td>
+      <td><code>String</code></td>
+      <td>
+        The content security policy used for content pages.
+      </td>
+    </tr>
```
Comment (mixedpuppy) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:82]: remove

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md [hunk 3]
```diff
@@ -59,18 +118,40 @@ There are restrictions on the policy you can specify here:
...
+
+```json
+"content_security_policy": "script-src 'self' https://example.com; object-src 'self'"
+```
+
+**Manifest V3**
+
+```json
+"content_security_policy": {
+  "content_scripts": "script-src 'self' https://example.com; object-src 'self'"
```
Comment (mixedpuppy) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:149]: change to an extension_page example

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md [hunk 4]
```diff
@@ -59,18 +118,40 @@ There are restrictions on the policy you can specify here:
...
 "content_security_policy": "default-src 'self'"
 ```
 
-Allow remote scripts from "https\://example.com":
+**Manifest V3**
 
 ```json
-"content_security_policy": "script-src 'self' https://example.com; object-src 'self'"
+"content_security_policy": {
+  "content_scripts": "default-src 'self'"
```
Comment (mixedpuppy) [inline files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:133]: change to extension_page

**Review by mixedpuppy — CHANGES_REQUESTED** [review]

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md (hunk 2 above)
Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:82]: Can I clarify, you made the content_script row?

Comment (mixedpuppy) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:82]: We added it at one point, then later removed it (which I forgot). Same with isolated_world

**Review by mixedpuppy — CHANGES_REQUESTED** [review]

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md [hunk 5]
```diff
@@ -50,6 +50,65 @@ There are restrictions on the policy you can specify here:
...
+          <li>Any localhost source, (<code>http://localhost</code>, <code>http://127.0.0.1</code>, or any port on those domains.)</li>
+        </ul>
+      </td>
+    </tr>
+    <tr>
+      <td><code>isolated_world</code></td>
+      <td><code>String</code></td>
+      <td>
+        An alias for <code>content_scripts</code> to provide Chrome compatibility. In Firefox, if both `isolated_world` and `content_scripts` are specified, the value from `content_scripts` is used.
+      </td>
```
[the remark this answers is not in the document]
Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:100]: Done

File: files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md (hunk 4 above)
Comment (rebloor) [reply files/en-us/mozilla/add-ons/webextensions/manifest.json/content_security_policy/index.md:133]: Done

**Review by mixedpuppy — APPROVED** [review]

**Pull request merged**